openid4vc-high-assurance-interoperability-profile-1_0.md

```diff
@@ -187,7 +187,7 @@ The following requirements apply to OpenID4VP, irrespective of the flow and Cred
 
 * The Wallet and Verifier MUST support at least one of the following Credential Format Profiles defined in (#vc-profiles): IETF SD-JWT VC or ISO mdoc. Ecosystems SHOULD clearly indicate which of these formats, IETF SD-JWT VC, ISO mdoc, or both, are required to be supported.
 * The Response type MUST be `vp_token`.
-* For signed requests, the Verifier MUST use, and the Wallet MUST accept the Client Identifier Prefix `x509_hash` as defined in Section 5.9.3 of [@!OIDF.OID4VP].
+* For signed requests, the Verifier MUST use, and the Wallet MUST accept the Client Identifier Prefix `x509_hash` as defined in Section 5.9.3 of [@!OIDF.OID4VP]. The X.509 certificate of the trust anchor MUST NOT be included in the `x5c` JOSE header of the signed request. The X.509 certificate signing the request MUST NOT be self-signed.
```

Comment:
There are another 2 places that we mention `x5c` and don't mention "The X.509 certificate of the trust anchor MUST NOT be included in the `x5c` JOSE header of the signed request" - should we just fix them all?

Reply:
I don't understand why we want to prohibit self signed certifcates. Issue #236 also does not explain the rationale. So I suggest to remove the second sentence.